frontend/src/style/card.scss

```diff
@@ -144,3 +145,70 @@
     }
   }
 }
+
+.service-card {
```

Comment:
Same as above, the class name is in conflict with one of my class names. I'll change mine so no problem here.